pandas/tests/io/test_excel.py

```diff
@@ -2490,3 +2499,71 @@ def custom_converter(css):
             n_cells += 1
 
     assert n_cells == (10 + 1) * (3 + 1)
+
```

Comment:
I would split the style tests off into another test file. you can in fact make a sub-dir ``excel`` and move ``test_excel.py`` there as well (not that if you do this, you need to add a ``__init__.py`` and add the dir to ``setup.py``.